### ISSUE TITLE
[Adaptive Cards: Select Host App Combo Box]: In scan mode, Screen reader focus is going twice on the list items under 'Host app' combo box.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8166]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requiste:**
1. Turn on the narrator using 'Win+ctrl+enter' key.
2. Activate the scan mode using 'caps+space' key.

**Repro Steps:**
1. Open the above URL in the browser.
2. Follow the above pre-requisite.
3. Navigate to 'Select host app' combo box using arrow keys.
4. Press enter key and activate the combo box.
5. Press down arrow key and navigate the list and observe the issue.

**Actual Result:**
While navigating the 'Select host app' combo box list options, on pressing down arrow key screen reader focus is landing on the hidden element and announcing as **'Blank'** and on pressing down arrow key again it is landing on the list item.

**Expected Result:**
In scan mode, screen reader focus should lands on the elements one time and it should not announce the information as blank.

**Note:**
1. Same issue is repro under Target Version combo box and 'Card Elements'.
2. Same issue is repro throughout the application for the combo box.

**User Impact:**
It will impact the screen reader user as they will get confused, and they have put extra effort to renavigate on the next control.

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/208636882-528f5e61-cfa2-4c57-b981-87d12db01381.mp4

### Card JSON

```JSON
NA
```
